taxi/views.py

```diff
@@ -87,3 +89,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
```

Comment:
you can remove ```all()``

Reply:
thx u a lot for ur comments . i fixed almost everything